proto/api.proto

```diff
@@ -1218,21 +1226,25 @@ message GetKlineRequest {
 }
 
 message GetKlineResponse {
-  string market = 1;
-  google.protobuf.Timestamp timestamp = 2;
-  repeated Candle candles = 3;
+  map<string, Candles> marketCandles = 1;
 }
 
 message Candle {
   google.protobuf.Timestamp startTime = 1;
   google.protobuf.Timestamp updateTime = 2;
-  double open = 3;
-  double close = 4;
-  double low = 5;
-  double high = 6;
-  double amount = 7;
-  double volume = 8;
-  double count = 9;
+  string marketAddress = 3;
+  Project project = 4;
```

Comment:
we either don't need it at all in V2 or make is a string and return the value we have in the DB instead of tieing to the Project enum again.